### DISCUSSION TITLE
FR: Sensors Interface

### DISCUSSION THREAD

**Discussion by darth-drew** [Security-Onion-Solutions/securityonion#2175] [Ideas] [answered]
It would be nice to be able to see the IP address of each sensor in the sensors interface. Can this be automated, and/or can the description field be made editable?

**Comment by jertel** [comment, marked as answer]
I can look into making it default to the primary IP. Making editable during setup would be nice too, but let's see about starting with the IP for now.

**Reply by jertel** [reply]
Tracking in #2178.
